CONTRIBUTING.md

```diff
@@ -22,38 +22,58 @@ To run the test suite, call `./test.sh` from your local machine - the script wil
 
 ## Testing
 
-### Running Tests
+### Running Unit Tests
 
-To run the service broker unit tests run:
+To run the Conjur Service Broker unit tests, run the following:
 ```sh-session
 $ # Build the base images and artifacts
 $ ./build.sh
 
-$ # Run the tests
-$ ./test.sh
+$ # Run the unit tests
+$ ./bin/test_unit
```

Comment:
Since these are 'copy-pastable commands', can we strike the `$`? Or even change the formatting to not use inline comments too. As a side note, if we use ` ```shell script`, Gobrain IDEs will provide syntax highlighting within the markdown file., but that's totally optional.

"'"
To run the Conjur Service Broker unit tests, first build the base image and artifacts:
```shell script
./build.sh
```

Then, run the tests with the following command:
```shell script
./bin/test_unit
```
"""